tests/ha/multi_pod_failure/test_multi_data_pod_failure.py

```diff
@@ -1296,101 +1288,80 @@ def test_continuous_reads_during_kpods_down(self):
         resp = self.ha_obj.check_s3bench_log(file_paths=fail_logs, pass_logs=False)
         assert_utils.assert_true(len(resp[1]) <= len(fail_logs),
                                  f"Logs which contain pass: {resp[1]}")
-        LOGGER.info("Step 7: Verified status for In-flight READs/Verify DI while %s (K) pods "
+        LOGGER.info("Step 4: Verified status for In-flight READs/Verify DI while %s (K) data pods "
                     "going down.", self.kvalue)
 
-        LOGGER.info("Step 8: Create multiple buckets and run IOs")
-        self.test_prefix = 'test-35773-1'
-        resp = self.ha_obj.ha_s3_workload_operation(s3userinfo=list(users.values())[0],
-                                                    log_prefix=self.test_prefix, nsamples=2,
-                                                    nclients=2)
+        if CMN_CFG["dtm0_disabled"]:
+            LOGGER.info("STEP 5: Create IAM user and perform WRITEs-READs-Verify-DELETEs with "
+                        "variable object sizes on degraded cluster")
+            users = self.mgnt_ops.create_account_users(nusers=1)
+            self.test_prefix = 'test-35773-1'
+            self.s3_clean.update(users)
+            resp = self.ha_obj.ha_s3_workload_operation(s3userinfo=list(users.values())[0],
+                                                        log_prefix=self.test_prefix,
+                                                        nsamples=2, nclients=2)
+        else:
```

Comment:
we can remove else. So if DTM disabled we will do READ/DI on new user/buckets and in both cases we can redo read/verify on existing healthy buckets

Reply:
This is last step where we are performing IOs. as we discussed yesterday. It is not about READs WRITEs or DELETEs.

Reply:
Sure but we need to do the else part anyways so thought that we might not need else condition